searchlib/src/tests/transactionlog/translogclient_test.cpp

```diff
@@ -409,8 +433,7 @@ visitDomainTest(TransLogClient & tls, Session * s1, const vespalib::string & nam
     auto visitor = tls.createVisitor(name, ca);
     ASSERT_TRUE(visitor);
     EXPECT_TRUE( visitor->visit(0, 1) );
-    for (size_t i(0); ! ca._eof && (i < 60000); i++ ) { std::this_thread::sleep_for(10ms); }
-    EXPECT_TRUE( ca._eof );
+    EXPECT_TRUE( ca.wait_for_eof() );
```

Comment:
Should these eof-waits be `ASSERT_TRUE` instead if it does not make sense to continue the test execution if they do not hold true?

Reply:
maybe, some were asserts, some were expects, I just left them as they were